packages/interface-internal/src/transport-manager.ts

```diff
@@ -2,13 +2,26 @@ import type { AbortOptions, Connection, Listener, Transport, TransportManagerDia
 import type { Multiaddr } from '@multiformats/multiaddr'
 import type { ProgressOptions } from 'progress-events'
 
+/**
+ * Options for dialing a connection using the `TransportManager`.
+ */
 export interface TransportManagerDialOptions extends AbortOptions, ProgressOptions<TransportManagerDialProgressEvents> {
 
 }
+/**
+ * The `TransportManager` module handles the management of transport protocols in a libp2p network.
+ * It is responsible for managing the transport themselves - dialling, querying addresses and listening.
+ */
+
+/**
+ * The `TransportManager` interface manages available transports for dialing and listening in a libp2p node.
+ */
 
```

Comment:
Must be a single comment, not two.

Also `protocols` is a loaded word here, since it is used to refer to protocol streams.  Better to just call transports "transports" or "network transports" and not "transport protocols".

Also, also, it's probably worth noting that consumers should generally prefer to open connections using the connection manager.

```suggestion

/**
 * The `TransportManager` handles the management of network transports, allowing
 * opening connections or listening using specific transports, etc.
 *
 * This is a low-level component - any connections opened will not be managed by
 * the `ConnectionManager` or included in any configured connection limits, etc.
 *
 * Most consumers will call `openConnection` on the `ConnectionManager` instead.
 */
```

Reply:
@achingbrain I've addressed the latest round of feedback—please let me know if there's anything else that needs further improvement.
I truly appreciate the time and effort you’ve put into reviewing my work and ensuring it meets the right standards :)